pkg/clusters/addons/kong/kong.go

```diff
@@ -146,11 +167,44 @@ func (a *Addon) Deploy(ctx context.Context, cluster clusters.Cluster) error {
 		)
 	}
 
+	if err := clusters.CreateNamespace(ctx, cluster, a.namespace); err != nil {
+		return err
+	}
+
 	// do the deployment and install the chart
 	args := []string{"--kubeconfig", kubeconfig.Name(), "install", DefaultDeploymentName, "kong/kong"}
-	args = append(args, "--create-namespace", "--namespace", a.namespace)
-	args = append(args, a.deployArgs...)
+	args = append(args, "--namespace", a.namespace)
+	if a.enterprise {
+		if a.license != "" && a.license == KongEnterpriseTestLicense {
+			if err := deployKongEnterpriseLicenseSecret(ctx, cluster, a.namespace, KongEnterpriseTestLicense); err != nil {
+				return err
+			}
+		}
+
+		if err := prepareSecrets(ctx, a.namespace); err != nil {
+			return err
+		}
+
+		args = append(args, "--version", "2.3.0", "-f", "https://raw.githubusercontent.com/Kong/charts/main/charts/kong/example-values/minimal-k4k8s-with-kong-enterprise.yaml")
```

Comment:
Could you please help me to understand better why we're forcing a specific version, only if enterprise is enabled?

What problem does this solve?

Reply:
we come up with the solution 
-  open a separate PR, persist the manifest content https://raw.githubusercontent.com/Kong/charts/main/charts/kong/example-values/minimal-k4k8s-with-kong-enterprise.yaml into KTF,  merge that PR and this PR will refer the default value from above file
-  keep the 2.3.0 version as it is